### DISCUSSION TITLE
Forcing specific wheel name

### DISCUSSION THREAD

**Discussion by lvjonok** [pypa/cibuildwheel#1653] [Q&A] [answered]
Good day!

I am trying to adapt cibuildwheel workflow to my project and all wheels after the build come with `py3-none` suffix which is not true :) I mean, I know that those wheels are definitely are dependent on python version and should be marked accordingly. Is there any option I can guide workflow to name them as `CIBW_BUILD` variable? For example, for `cp310-manylinux` it will create `*-cp310-cp310-*` wheel.

My workflow is [here](https://github.com/lvjonok/cmeel-casadi/blob/feat/cibuildwheel/.github/workflows/release.yml) if it would be helpful somehow.

**Comment by henryiii** [comment, marked as answer]
The wheel name is up to the build backend. You are using cmeel, which seems to be a rather exotic CMake based backend that was designed (as far as I can tell) to ship binaries and not Python extensions. It intentionally installs things into folders you can't import in Python. I'd recommend trying scikit-build-core if you need a Python extension with CMake.

**Reply by lvjonok** [reply]
Thank you!
My choice of cmeel is guided by other libraries I depend on. I think your answer *it is up to the build backend* suits me.

**Reply by henryiii** [reply]
~~Let me know if I'm wrong, but as far as I can tell, cmeel wasn't designed to allow you to make something importable.~~ Ahh, no, I see "enable Python binding" in the example. So there must be some way to install into `site-packages/<something>`. Okay, then I bet there's a configuration flag somewhere, or it's a bug.
